pkg/oras/settings_test.go

```diff
@@ -0,0 +1,93 @@
+package oras
+
+import "testing"
```

Comment:
Nice! I think we need to add a command in the `Makefile` for these to run, and run it in the main.yaml CI?

Reply:
Long story (but this is because of how the OCI layout doesn't have a contect of a tag)